src/components/ArticleCard/index.tsx

```diff
@@ -32,7 +32,12 @@ const ArticleCard = ({
   const { t } = useTranslation(translation);
   const history = useHistory();
 
-  const clickHandler = (url: string) => {
+  const clickHandler = (e: React.MouseEvent<HTMLElement>, url: string) => {
+    const target = e.target as Element;
+    if (target.getAttribute('data-testid') === 'tag') {
+      // TODO: history.push to url specified in tag
```

Comment:
@patrickseguraoddball 
I've updated the click handler for the `ArticleCard` component to now redirect if clicked anywhere but the tag. 
I also left a TODO comment to further update the card to take in the tag's url

Reply:
IIRC the tags don't need a url, just a type.  Just being able to click on the tag, the component will infer the route by the type

Reply:
But that might be a part of Ashley's shared work if it wasn't in your branch yet

Reply:
After I made that comment, I realized that it was in the AC to make sure the tags work, so I updated it the code.

Yup, you are right in that it does not a url, but just take account of the tag type. My updated code addresses that